services/src/northbound/src/main/java/org/openkilda/northbound/controller/HealthCheckController.java

```diff
@@ -71,11 +66,10 @@
     @RequestMapping(value = "/health-check",
             method = RequestMethod.GET,
             produces = MediaType.APPLICATION_JSON_UTF8_VALUE)
-    public ResponseEntity<HealthCheck> getHealthCheck(
-            @RequestHeader(value = CORRELATION_ID, defaultValue = DEFAULT_CORRELATION_ID) String correlationId) {
+    public ResponseEntity<HealthCheck> getHealthCheck() {
         logger.debug("getHealthCheck");
 
-        HealthCheck healthCheck = healthCheckService.getHealthCheck(correlationId);
+        HealthCheck healthCheck = healthCheckService.getHealthCheck(RequestCorrelation.getId());
```

Comment:
Why don't we read the correlationId inside the service as it's done in other resources (flowController, etc.)?